ratis-grpc/src/main/java/org/apache/ratis/grpc/metrics/MessageMetrics.java

```diff
@@ -17,49 +17,71 @@
  */
 package org.apache.ratis.grpc.metrics;
 
+import org.apache.ratis.metrics.LongCounter;
 import org.apache.ratis.metrics.MetricRegistryInfo;
 import org.apache.ratis.metrics.RatisMetrics;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+
+import java.util.Map;
 
 public class MessageMetrics extends RatisMetrics {
-  static final Logger LOG = LoggerFactory.getLogger(MessageMetrics.class);
   public static final String GRPC_MESSAGE_METRICS = "%s_message_metrics";
   public static final String GRPC_MESSAGE_METRICS_DESC = "Outbound/Inbound message counters";
 
+  private enum Type {
+    STARTED("_started_total"),
+    COMPLETED("_completed_total"),
+    RECEIVED("_received_executed");
```

Comment:
@codings-dan , I also want to change some of the metrics names but it may break some external metrics applications.  So, we are better to leave them unchanged.